src/scheduler/fifo.h

```diff
@@ -203,7 +203,7 @@ int add_job_to_calendar(int pbs_sd, status *policy, server_info *sinfo, resource
  *	       first move it to the local server and then run it.
  *	       if it's a local job, just run it.
  */
-int run_job(int pbs_sd, resource_resv *rjob, char *execvnode, int had_runjob_hook, schd_error *err);
+int run_job(int pbs_sd, resource_resv *rjob, char *execvnode, int had_runjob_hook, char *, schd_error *err);
```

Comment:
Add the name of the variable to char * like the others

Reply:
Ah, I missed it. added now.